reducers/studyspacesReducer.js

```diff
@@ -45,7 +55,7 @@ export default (state = initialState, action = null) => {
       const newStudyspaces = ids.reduce(
         (spaces, s) =>
           updateStudyspaces(spaces, s, {
-            ...state.studyspaces.filter(x => x.id === s),
+            ...state.studyspaces.filter(x => x.id === s)[0],
```

Comment:
Fixed what appears to have been a typo here. The functionality of the app wasn't affected because the data in question was re-fetched and replaced with the correct data in the StudySpaceDetailScreen (before it was ever displayed).

We should probably have unit tests at some point, especially for redux.